### PR TITLE
LPS-178473 Fix Feature Flag ticket

### PR DESCRIPTION
**Pull request by marcelabc**
Related issue: https://issues.liferay.com/browse/LPS-178473

cc: @DaneszLR

### CODE REVIEW

**Comment by marcelabc** [comment]
Hey @carolmariaabb,
Could you review my changes, please?

File: modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/graphql/dto/v1_0/ObjectDefinitionGraphQLDTOContributor.java [hunk 1]
```diff
@@ -357,38 +404,85 @@ private ObjectDefinitionGraphQLDTOContributor(
...
 		_objectDefinition = objectDefinition;
+		_objectDefinitionLocalService = objectDefinitionLocalService;
 		_objectEntryManager = objectEntryManager;
+		_objectRelationshipLocalService = objectRelationshipLocalService;
 		_objectScopeProvider = objectScopeProvider;
 		_relationshipGraphQLDTOProperties = relationshipGraphQLDTOProperties;
 		_resourceName = resourceName;
 		_typeName = typeName;
+		_systemObjectDefinitionManagerRegistry =
+			systemObjectDefinitionManagerRegistry;
```
Comment (carolmariaabb) [inline modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/graphql/dto/v1_0/ObjectDefinitionGraphQLDTOContributor.java:430]: @marcelabc same here, `_systemObjectDefinitionManagerRegistry` before `_typeName`

File: modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/graphql/dto/v1_0/ObjectDefinitionGraphQLDTOContributor.java [hunk 2]
```diff
@@ -357,38 +404,85 @@ private ObjectDefinitionGraphQLDTOContributor(
...
+		_extensionProviderRegistry = extensionProviderRegistry;
```
Comment (carolmariaabb) [inline modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/graphql/dto/v1_0/ObjectDefinitionGraphQLDTOContributor.java:431]: @marcelabc same here, `_extensionProviderRegistry` before `_graphQLDTOProperties`

File: modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/graphql/dto/v1_0/ObjectDefinitionGraphQLDTOContributor.java [hunk 3]
```diff
@@ -357,38 +404,85 @@ private ObjectDefinitionGraphQLDTOContributor(
...
 
-		return 0;
+		return null;
+	}
+
+	private ExtendedEntity _toExtendedEntity(
+			DTOConverterContext dtoConverterContext,
+			ObjectDefinition objectDefinition,
+			SystemObjectDefinitionManager systemObjectDefinitionManager,
+			long relatedObjectEntryId)
```
Comment (carolmariaabb) [inline modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/graphql/dto/v1_0/ObjectDefinitionGraphQLDTOContributor.java:456]: @marcelabc same here, `relatedObjectEntryId` before `systemObjectDefinitionManager`

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

**Review by carolmariaabb — APPROVED** [review]
LGTM!

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:forward:force

**Pull request closed without merging**